src/components/input/input-short/InputShort.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import { errorMessageCSS, InputShortComponentCSS } from "./InputShortCSS";
+import { colors } from "../../colors";
+const getDisabledOrErrorStyles = (p) => {
+  if (p.disabled) {
+    return `border-color: ${colors.grey500}; color:${colors.grey500}; &:hover {
+      border-color: ${colors.grey500};
+    }`;
+  }
+  if (p.error) {
+    return `border-color: ${colors.red500};  &:hover {
+      border-color: ${colors.red500};
+    }
+    &:focus {
+      color: white;
+    }
+    `;
+  }
+};
+
+const InputShortComponent = styled.input`
+  ${InputShortComponentCSS};
+  ${(p) => getDisabledOrErrorStyles(p)}
+`;
+
+const ErrorMessage = styled.div`
+  ${errorMessageCSS};
+`;
+
+const InputShort = ({ disabled, error, errorMessage }) => {
+  return (
+    <div>
+      <InputShortComponent
+        id="input-short-component"
+        disabled={disabled}
+        error={error}
+      />
+      {error && !disabled && <ErrorMessage>{errorMessage}</ErrorMessage>}
```

Comment:
Will there eventually be logic to produce an error message if the text is too long for the short answer portion? Just wondering if that logic should be included here

Reply:
I didn't initially add it as there wasn't a design for it, but I do think we need error handling for InputShort as well. I've added error handling if the text is too long